service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/legalcounsel/LegalCounsellorRemovedNotifyTemplate.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.model.notify.legalcounsel;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import uk.gov.hmcts.reform.fpl.model.notify.SharedNotifyTemplate;
+
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@Data
+@SuperBuilder
+public class LegalCounsellorRemovedNotifyTemplate extends SharedNotifyTemplate {
```

Comment:
do these need to extend this template? do the notify template use all of those?

Reply:
same for other one

Reply:
I doesn't, but it uses one or two.
Because it just ignores everything else, I thought it was better to do this than to create duplicate attributes.
I don't mind changing this if you prefer it with the duplication. Let me know.

Reply:
Fair enough. Will change that.